emulator.c

```diff
@@ -52,8 +75,19 @@ execute_instruction(i8080 *cpu, uint8_t opcode)
       printf("DAD D");
       break;
     case 0x1a: // NOLINT
-      printf("LDAX D");
-      break;
+      {        // LDAX D
+               // get addr
+        uint16_t addr = cpu->e;
+        addr = addr << 8; // NOLINT
+        addr += cpu->d;
```

Comment:
Is this the right order? the 16-bit address should be DE but you have it as ED? I think Little Endian would only be a thing when we are trying to get an address from popping off the stack. I think register D holds bits 8-15 and E 0-7 when building a 16-bit value.

Reply:
I think this true here and in the other cases below. I'll update this and the unit tests, hopefully get it done sometime today.